--- pytest_asyncio-0.26.0/pyproject.toml.orig	2025-03-25 07:19:03.000000000 +0100
+++ pytest_asyncio-0.26.0/pyproject.toml	2025-03-27 23:45:00.176648808 +0100
@@ -10,10 +10,6 @@
 description = "Pytest support for asyncio"
 readme.content-type = "text/x-rst"
 readme.file = "README.rst"
-license = "Apache-2.0"
-license-files = [
-  "LICENSE",
-]
 authors = [
   { name = "Tin Tvrtković <tinchester@gmail.com>", email = "tinchester@gmail.com" },
 ]
